include/LightGBM/utils/text_reader.h

```diff
@@ -119,6 +120,15 @@ class TextReader {
       if (last_i != read_cnt) {
         last_line_.append(buffer_process + last_i, read_cnt - last_i);
       }
+
+      size_t prev_bytes_read = bytes_read;
+      bytes_read += read_cnt;
+      if (prev_bytes_read / read_progress_interval_bytes_ < bytes_read / read_progress_interval_bytes_)
```

Comment:
I am a little bit confused about this `if`. 
maybe `prev_bytes_read` should be in this `if` block?

Reply:
This is just checking if the integer value after dividing by read_progress_interval_bytes_ changed before and after reading the current block of data. So if the previous bytes read was 9.9GB, and now it is 10.1GB, we will print a log line.